core/DependencyInjection/ContainerBuilder.php

```diff
@@ -97,19 +97,60 @@ public function forceBuild(): ContainerInterface
         );
         $phpLoader->load('services.php');
 
-        $legacyLoader = new LegacyFileLoader(
-            $this,
-            new FileLocator(
-                [
-                    $this->configPath
-                ]
-            )
-        );
-        $legacyLoader->load('*/*.conf.php');
-
         return $this->cache->forceLoad();
     }
 
+    /**
+     * @inheritDoc
+     */
+    public function get(string $id, int $invalidBehavior = SynfonyContainerInterface::EXCEPTION_ON_INVALID_REFERENCE)
+    {
+        try {
+            return parent::get($id, $invalidBehavior);
+        } catch (SymfonyServiceNotFoundException $exception) {
+        }
+
+        try {
+            return $this->legacyContainer->get($id);
+        } catch (ServiceNotFoundException $exception) {
+            throw new SymfonyServiceNotFoundException($id);
```

Comment:
Should it leave some traces in logs for an debugging purposes?

Reply:
I this case IMO it is not necessary, since this is the fallback for all Legacy services, so we are expected to have this 99% of the time.